my-app/src/location.js

```diff
@@ -1,23 +1,25 @@
 export function geoFindMe() {
-    const status = document.querySelector('#status');
-    const mapLink = document.querySelector('#map-link');
-    mapLink.href = '';
-    mapLink.textContent = '';
+  var latitude;
+  var longitude;
     function success(position) {
-      const latitude  = position.coords.latitude;
-      const longitude = position.coords.longitude;
-      status.textContent = '';
-      mapLink.href = `https://www.openstreetmap.org/#map=18/${latitude}/${longitude}`;
-      mapLink.textContent = `Latitude: ${latitude} °, Longitude: ${longitude} °`;
+      latitude  = position.coords.latitude;
+      longitude = position.coords.longitude;
+      // status.textContent = '';
```

Comment:
Minor: the comments can be removed

Reply:
Done, will send second revision after all comments are done